webapp/src/main/resources/locale/portlet/Login_fr.properties

```diff
@@ -1,5 +1,6 @@
 UILoginForm.msg.Invalid-account=Nom d'utilisateur et/ou mot de passe incorrect ou manquant. Veuillez ré-essayer.
 loginForm.settings.editTooltip=Editer les param\u00E8tres
+loginForm.settings.back=Retour \u00E0 la page pr\u00E9c\u00E9dente
```

Comment:
why do we need to encode the special chars here ?

Reply:
When I type a translated string into my IDE, I always use encoded char to prevent file encoding pb : sometimes files are UTF-8 encoded, somethimes ISO-8859-1

So, to be sure I encode.

At the end, when the string is translated in crowdin, the file is always correctly encoded, so it does not really matters.